uncrustify.cfg

```diff
@@ -60,19 +60,25 @@ sp_cmt_cpp_start			= add
 sp_cmt_cpp_doxygen			= true
 sp_angle_shift				= remove
 sp_permit_cpp11_shift		= true
+sp_before_sparen			= true
```

Comment:
[This](https://github.com/uncrustify/uncrustify/blob/master/documentation/htdocs/config.txt) shows that the possible values are `Ignore, Add, Remove, Force`. Not 'true'